webserver/views/api/v1/test/test_core.py

```diff
@@ -118,7 +118,7 @@ def test_ll_no_item(self, ll):
     @mock.patch("db.data.load_high_level")
     def test_hl_numerical_offset(self, hl):
         hl.return_value = {}
-        resp = self.client.get("/api/v1/%s/high-level?n=3" % self.uuid)
+        resp = self.client.get("/api/v1/%s/high-level?n=3&map_model_class_names=False" % self.uuid)
```

Comment:
You shouldn't need to modify these tests, since the default is to treat the parameter as False.

Instead, we should have a separate test which checks that the right thing is one when this parameter is set to true.